components/http_kernel.rst

```diff
@@ -759,10 +759,10 @@ directory of AppBundle.
 The HttpKernel component provides a method called :method:`Symfony\\Component\\HttpKernel\\Kernel::locateResource`
 which can be used to transform logical paths into physical paths::
 
-    use Symfony\Component\HttpKernel\HttpKernel;
+    use Symfony\Component\HttpKernel\Kernel;
 
     // ...
-    $kernel = new HttpKernel($dispatcher, $resolver);
+    $kernel = new Kernel($dispatcher, $resolver);
```

Comment:
that cannot be correct: Kernel doesn't take these as arguments

Reply:
Hmm you are right... 🤔

Reply:
The more I look at this, the more problems I see with my request. Maybe this code example should be rewritten from scratch? I don't really see anybody creating a new `Kernel` instance just to get a path to a config file.